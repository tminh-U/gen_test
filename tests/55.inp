10
2 3 19 3 3 12 6 16 17 5 
15
7 10
2 2
4 4
3 4
10 10
4 8
2 9
10 10
5 7
6 10
1 7
3 6
10 10
10 10
4 10
