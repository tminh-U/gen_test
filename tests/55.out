9
2
2
3
2
24
98
2
6
13
52
9
2
2
52
